index.html

```diff
@@ -100,8 +100,11 @@
           <span><input type="checkbox" id="isNoteworthy" name="isNoteworthy"></span>
         </div>
       </div>
-      <div class="inputBox">
+      <div class="inputBox loaderContainer">
```

Comment:
Do we need to have a separate `loaderContainer` class?

Reply:
no. Actually the only thing required here is to add `position: realtive` style. it can be added in `inputBox` class but it's no recommended to modify the `inputBox`  class.